department-of-veterans-affairs/vets-api
# [56518] - Disable Rails console autocomplete

## Summary

- Disable Rails console autocomplete in deployed environments to prevent running unintended commands. 

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#56518


## Testing done

- remove conditional in `.irbrc` or change to `ENV['RAILS_ENV'] == 'development'`
- run `make console`
- autocomplete should be disabled in the console


## What areas of the site does it impact?
Rails console in deployed environments

## Acceptance criteria
